lib/events.ts

```diff
@@ -1,43 +1,51 @@
 import * as Capitano from 'capitano';
 
 import _ = require('lodash');
-import Mixpanel = require('mixpanel');
-import Raven = require('raven');
+import {
+	core as Analytics,
+	mixpanelIntegration,
+	sentryIntegration,
+} from 'analytics.node';
 import Promise = require('bluebird');
 import ResinSdk = require('resin-sdk');
 import packageJSON = require('../package.json');
+Analytics.addIntegration(sentryIntegration);
 
 const resin = ResinSdk.fromSharedOptions();
 const getMatchCommandAsync = Promise.promisify(Capitano.state.getMatchCommand);
-const getMixpanel = _.memoize<any>(() =>
+const getAnalyticsInstance = _.memoize<any>(() =>
 	resin.models.config
 		.getAll()
 		.get('mixpanelToken')
-		.then(Mixpanel.init),
+		.then(token => {
+			Analytics.addIntegration(mixpanelIntegration);
+			let options = { token: token };
+			return Analytics.initialize({ Mixpanel: options });
+		}),
 );
 
 export function trackCommand(capitanoCli: Capitano.Cli) {
 	return Promise.props({
 		resinUrl: resin.settings.get('resinUrl'),
 		username: resin.auth.whoami().catchReturn(undefined),
-		mixpanel: getMixpanel(),
+		analytics: getAnalyticsInstance(),
 	})
-		.then(({ username, resinUrl, mixpanel }) => {
+		.then(({ username, resinUrl, analytics }) => {
 			return getMatchCommandAsync(capitanoCli.command).then(command => {
-				Raven.mergeContext({
+				Analytics.mergeContext({
```

Comment:
Is there any documentation anywhere on what `initialize` does, and what you're supposed to call on the global vs calling on the returned instance? I'm a little unclear about the structure of that, maybe if I understood that better this would be clearer.

Reply:
yeah, I see your point. I'll change the names for now.
Regarding the other comment, initialize() calls the init/install method of the desired integration.